faunadb/request_response_info.py

```diff
@@ -0,0 +1,45 @@
+class RequestResponseInfo(object):
```

Comment:
Do we want to make use of these classes internally to pass requests and responses around, or just for the observer?